client/src/components/content/ModelerSidebar/PropertiesPanel/PropertiesPanelComponents/PPParameterInput.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Input, Tooltip } from 'antd';
+import { InfoCircleOutlined } from '@ant-design/icons';
+import PropTypes from 'prop-types'
+import corporateDesign from '../../../../../layout/corporateDesign';
+
+/**
+ * @description Renders the task-dropdown based on passed list of task.
+ * @category Client
+ * @component
+ */
+const PPParameterInput = ({
+    onValueChange,
+    variableName,
+    isRequired,
+    // eslint-disable-next-line no-unused-vars
+    dataType,
```

Comment:
Why do we have dataType as input, although it's not used?

Reply:
This is passed in already because it future proofs this component when different fields can be displayed for each variable